retis-events/src/skb.rs

```diff
@@ -145,7 +145,7 @@ impl EventFmt for SkbEvent {
 
         if let Some(packet) = &self.packet {
             space.write(f)?;
-            packet.packet.event_fmt(f, format)?;
+            packet.raw.event_fmt(f, format)?;
```

Comment:
How about moving it to its own section instead and keep skb as metadata-only?
we could have packets in userspace or xdp without an skb so this would be a step in the right direction.

Reply:
Let's discuss this in the other related comment.